app/src/main/java/com/github/se/gomeet/ui/navigation/BottomNavigationMenu.kt

```diff
@@ -18,17 +19,21 @@ fun BottomNavigationMenu(
     tabList: List<TopLevelDestination>,
     selectedItem: String
 ) {
-  BottomNavigation(
-      modifier = Modifier.fillMaxWidth().height(80.dp),
-      backgroundColor = MaterialTheme.colors.background) {
+  NavigationBar(
+      modifier = Modifier.height(80.dp),
+      containerColor = MaterialTheme.colorScheme.surfaceVariant) {
         tabList.forEach { destination ->
-          BottomNavigationItem(
+          NavigationBarItem(
               icon = {
-                Icon(imageVector = destination.icon, contentDescription = destination.textId)
+                Icon(
+                    imageVector = getIconForRoute(destination.route),
```

Comment:
I do not see the point of the function getIconForRoute. Top level destinations already have an attribute icon